frontend/src/static/js/main.js

```diff
@@ -72,6 +75,7 @@ window.app = new window.Vue({
       });
       return full;
     },
+
```

Comment:
All the changes in this file here seems unnecessary.

Reply:
yeah, just some minor cleanups i missed before that

Reply:
> yeah, just some minor cleanups i missed before that

Such changes should only be done if the changes touch file or in a separate PR. 
Otherwise it will complicate things should we need to revert the PR in future.

Reply:
i removed the part for the empty state already, so i removed the alignment as well?